index.js

```diff
@@ -405,4 +430,13 @@ function createUserArray() {
 }
 function hash(str) {
 	return crypto.createHash("sha256").update(str).digest("hex");
+}
+function isUserAdmin(name) {
+	if(!config) {
+		return false;
+	}
+	if(!config.hasOwnProperty("adminuser")) {
+		return false;
+	}
+	return config.adminuser.includes(hash(name));
```

Comment:
suggest following:
```
function isUserAdmin(name) {
if(!config) {
    return false;
}
return config.adminuser?.includes(hash(name)) || false;
}

```
When adminuser is not set `config.adminuser?` will return `undefined`. `undefined  || false` will return false.